packages/instant/src/util/address.ts

```diff
@@ -0,0 +1,6 @@
+import { web3Wrapper } from '../util/web3_wrapper';
+
+export const getBestAddress = async (): Promise<string> => {
+    const addresses = await web3Wrapper.getAvailableAddressesAsync();
+    return addresses[0];
```

Comment:
what happens if `addresses` is empty

Reply:
is this a possibility?  if so, I could have this thrown and flash an error

Reply:
This should be handled by `hasSufficientFunds` returning `false`

Reply:
Ok, I was playing around and a locked metamask will produce an empty array here so we should handle it by flashing an error

Reply:
Ah cool, didn't know this is what would happen when metamask was is locked. It will show "Not enough ETH" for now, I wonder if we should make a better message in a future PR?

Reply:
yea let me make a ticket for that